12/umbraco-vendr/core/how-to-guides/migrating-from-vendr-to-umbraco-commerce.md

```diff
@@ -0,0 +1,319 @@
+WHERE textValue LIKE '%Vendr.VariantsEditor%';
+```
+
+5. Swap Vendr price/amount adjustments to Umbraco Commerce price/amount adjustments:
+
+```sql
+UPDATE umbracoCommerceOrderPriceAdjustment
+SET type = REPLACE(type, 'Vendr.', 'Umbraco.Commerce.')
+WHERE type LIKE '%Vendr.%';
+UPDATE umbracoCommerceOrderAmountAdjustment
+SET type = REPLACE(type, 'Vendr.', 'Umbraco.Commerce.')
+WHERE type LIKE '%Vendr.%';
+```
+
+6. Update template paths:
+
+```sql
+UPDATE umbracoCommerceEmailTemplate
+SET templateView = REPLACE(templateView, '/App_Plugins/Vendr/templates/email', '/Views/UmbracoCommerce/Templates/Email')
+WHERE templateView LIKE '%/Vendr/%';
+UPDATE umbracoCommercePrintTemplate
+SET templateView = REPLACE(templateView, '/App_Plugins/Vendr/templates/print', '/Views/UmbracoCommerce/Templates/Print')
+WHERE templateView LIKE '%/Vendr/%';
+UPDATE umbracoCommerceExportTemplate
+SET templateView = REPLACE(templateView, '/App_Plugins/Vendr/templates/email', '/Views/UmbracoCommerce/Templates/Export')
+WHERE templateView LIKE '%/Vendr/%';
+```
+
+7. Update the migrations log:
+
+```sql
+UPDATE umbracoCommerceMigrations
+SET migration = REPLACE(migration, 'Vendr.', 'Umbraco.Commerce.')
+WHERE migration LIKE 'Vendr.%';
+```
+
+8. Update the activity logs:
+
+```sql
+UPDATE umbracoCommerceActivityLog
+SET eventType = REPLACE(eventType, 'vendr/', 'commerce/')
+WHERE eventType LIKE 'vendr/%';
+```
+
+### Step 4: Finalizing the migration
+
+1. Delete any obj/bin folders in your projects to ensure a clean build.
+2. Recompile all projects and ensure all dependencies are restored correctly
+3. Change license files - TBD
+4. Update any payment gateways that uses a global webhook:
+
+```none
+https://{site_url}/umbraco/commerce/payment/callback/{payment_provider_alias}/{payment_method_id}/
+```
+
+5. Run the project.
+
+Before moving on to migrating the pakcages and payment providers, it is recommended to ensure everything works as expected.
+
+## Migrate Umbraco Commerce Checkout
+
+Throught the following steps we will migrate the Checkout package from Vendr to Umbraco Commerce.
+
+### Step 1: Backup
+
+1. Make a backup of any custom templates and Umbraco Commerce UI configuration files.
+2. Make a note of all configuration values on the Vendr.Checkout checkout node.
+
+### Step 2: Uinstall Vendr.Checkout
+
+1. Delete Vendr.Checkout generated checkout nodes.
+2. Delete all Vendr.Checkout generated Document Types.
+3. Delete all Vendr.Checkout generated Data Types.
+4. Uninstall the Vendr.Checkout nuget package:
+
+```bash
+dotnet remove package Vendr.Checkout
+```
+
+5. Delete any remaining `~/App_Plugins/VendrCheckout` files and folders.
+
+### Step 3: Install Umbraco.Commerce.Checkout
+
+1. Install the Umbraco.Commerce.Checkout package
+
+    ```bash
+    dotnet add package Umbraco.Commerce.Checkout
+    ```
+
+2. In the settings section, locate the Umbraco Commerce Checkout dashboard and reinstall into previous location.
```

Comment:
I'm not sure I understand what this means. Is it something that you need to install in the backoffice?

Reply:
Ah!
Then let's add item #3 here:

2. Locate the Umbraco Commerce Checkout dashboard in the Settings section.
3. Click the "install" button to reinstall the Checkout components into the previous location.

What do you think?

Reply:
Yup, I think that would be perfect 👍